monarch_ingest/alliance/gene2disease.yaml

```diff
@@ -45,3 +47,5 @@ edge_properties:
   - 'negated'
   - 'publications'
   - 'provided_by'
+
+transform_mode: 'flat'
```

Comment:
I meant to flip this back so it's the default, but it's good to be explicit anyways